runtime/common/src/evm/precompile.rs

```diff
@@ -94,11 +99,75 @@ where
 	}
 }
 
+<<<<<<< HEAD
+/// A set of precompiles. This set might contain
+/// not yet mainnet ready precompiles in order to test
+/// those in development or staging environment without touching
+/// the mainnet set.
+pub struct Development<R>(PhantomData<R>);
+
+impl<R> Development<R> {
+	#[allow(clippy::new_without_default)] // We'll never use Default and can't derive it.
+	pub fn new() -> Self {
+		Self(Default::default())
+	}
+}
+
+impl<R> PrecompileSet for Development<R>
+where
+	R: pallet_evm::Config + axelar_gateway_precompile::Config + frame_system::Config,
+	R::RuntimeCall: Dispatchable<PostInfo = PostDispatchInfo> + GetDispatchInfo + Decode,
+	<R::RuntimeCall as Dispatchable>::RuntimeOrigin: From<Option<R::AccountId>>,
+	axelar_gateway_precompile::Pallet<R>: Precompile,
+{
+	fn execute(&self, handle: &mut impl PrecompileHandle) -> Option<PrecompileResult> {
+		match handle.code_address().0 {
+			ECRECOVER_ADDR => Some(ECRecover::execute(handle)),
+			SHA256_ADDR => Some(Sha256::execute(handle)),
+			RIPEMD160_ADDR => Some(Ripemd160::execute(handle)),
+			IDENTITY_ADDR => Some(Identity::execute(handle)),
+			MODEXP_ADDR => Some(Modexp::execute(handle)),
+			BN128ADD_ADDR => Some(Bn128Add::execute(handle)),
+			BN128MUL_ADDR => Some(Bn128Mul::execute(handle)),
+			BN128PAIRING_ADDR => Some(Bn128Pairing::execute(handle)),
+			BLAKE2F_ADDR => Some(Blake2F::execute(handle)),
+			SHA3FIPS256_ADDR => Some(Sha3FIPS256::execute(handle)),
+			DISPATCH_ADDR => Some(Dispatch::<R>::execute(handle)),
+			ECRECOVERPUBLICKEY_ADDR => Some(ECRecoverPublicKey::execute(handle)),
+			CONNECTORS_AXELAR_GATEWAY => Some(<axelar_gateway_precompile::Pallet<R> as Precompile>::execute(handle)),
+			_ => None,
+		}
+	}
+
+	fn is_precompile(&self, address: H160) -> bool {
+		[
```

Comment:
As above. Bools have the same `.or_else` combinator as Options now, so the logic would be nearly identical.